classes/02class/exercises/c02-network01/marcio-anatrielo/c02-network01/c02-network01.txt

```diff
@@ -0,0 +1 @@
+123
```

Comment:
Please, commit the real answer so we can review and merge it.

Reply:
# Description

Exercise (C02-NETWORK01): 

Student Name: Marcio Andre

> A brief explanation of what has been done.

Step 1: Check IP Calculator 

Step 2: check IP Whois 


## Type of change

- [X] Exercise Submission (one exercise per PR)
  - [X] This PR is just for **one** exercise of a class
  - [X] PR name follows the pattern `<github-username>/<exercise-number>`
  - [X] Added exercise files inside folder `/classes/<class- number>/exercises/<exercise-number>/<github-username>/`
	- [X] Review [exercise submission steps](./README.md#Exercises)
- [ ] Class content (instructors/admins)
- [ ] Documentation update
- [ ] Repo management